### PR TITLE
feature/BAK 1260/change ci publish

### PR DESCRIPTION
**Pull request by diversemix**
Test branch to get pushing images to new prod - and honouring semver

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -35,6 +30,8 @@ jobs:
         run: |
           docker-compose build
           docker images
+          docker tag navigator_backend navigator-backend
```
Comment (diversemix) [inline .github/workflows/ci.yml:33]: @joel-wright / @henry-cpr ... I think this is the only point of contention - happy to change the name of the ECR registries to have underscores (not hyphens) - but my preference would be to change the `docker-compose.yml`

File: backend/app/db/crud/document.py [hunk 2]
```diff
@@ -732,7 +732,7 @@ def create_document_relationship(
         # The call to db.commit() must exist outside the nested transaction
         db.commit()
     except IntegrityError:
-        # ensure idemopotency
+        # ensure impotency
```
Comment (diversemix) [inline backend/app/db/crud/document.py:735]: oops

Comment (joel-wright) [reply backend/app/db/crud/document.py:735]: don't do that!!

Comment (henry-cpr) [reply backend/app/db/crud/document.py]: Possibly one of my favourite typos I've seen in a PR

File: .github/workflows/ci.yml (hunk 1 above)
Comment (henry-cpr) [reply .github/workflows/ci.yml:33]: Can you take me through the either/or thinking and what changes are involved?

**Review by henry-cpr — APPROVED** [review]

**Comment by diversemix** [comment]
Follow up changes to come - see PR

**Pull request merged**